cmd/quota_test.go

```diff
@@ -0,0 +1,15 @@
+package cmd
+
+import "testing"
+
+func TestQuotaNil(t *testing.T) {
```

Comment:
I am just leaving a note here that if you split the status into 3 statuses we would also need more tests :)